vms/platformvm/utxo/handler.go

```diff
@@ -69,7 +69,7 @@ func Produce(
 }
 
 // TODO: Stake and Authorize should be replaced by similar methods in the
-//	P-chain wallet
+//       P-chain wallet
```

Comment:
thats actually revert to avax current branch (compared with 481dee9c3bd2bb1b5dd988e054668f0c999b3e81 commit)